samples/bluetooth/ipsp/README.rst

```diff
@@ -19,22 +19,164 @@ Testing with a Linux host
 =========================
 
 To test IPSP please take a look at samples/net/README, in addition to running
-echo-client you must enable 6LowPAN module in Linux with the
-following commands:
+echo-client you must enable 6LowPAN module in Linux with the following commands
+(as root):
 
 .. code-block:: console
 
-   $ modprobe bluetooth_6lowpan
-   $ echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
+   # modprobe bluetooth_6lowpan
+   # echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
 
-Then to connect:
+If you connected your board to a UART console, you will see an output similar to
+(may vary slightly by application and Zephyr versions):
 
 .. code-block:: console
 
-   $ echo "connect <bdaddr> <type>" > /sys/kernel/debug/bluetooth/6lowpan_control
+   [bt] [WRN] set_static_addr: Using temporary static random address
+   [bt] [INF] show_dev_info: Identity: cb:af:14:57:d8:6e (random)
+   [bt] [INF] show_dev_info: HCI: version 5.0 (0x09) revision 0x0000, manufacturer 0xffff
+   [bt] [INF] show_dev_info: LMP: version 5.0 (0x09) subver 0xffff
+   [bt] [WRN] bt_pub_key_gen: ECC HCI commands not available
+   [ipsp] [INF] init_app: Run IPSP sample
+   [ipsp] [INF] listen: Starting to wait
 
-Once connected a dedicated interface will be created, usually bt0, which can
-then be used as following:
+The output above shows the BLE address assigned to your board for the
+current session; the address will be different on subsequent sessions.
+
+Alternatively, you may scan for your board on the host. The modern way to do
+that is using ``bluetoothctl`` utility (included in the recent versions of
+BlueZ package) and its ``scan on`` command:
+
+.. code-block:: console
+
+   $ bluetoothctl
+   [NEW] Controller A3:24:97:EB:D6:23 ubuntu-0 [default]
+   [NEW] Device D7:5C:D6:18:14:87 Zephyr
+   [NEW] Device E1:E7:F9:56:EC:06 Zephyr
+   [NEW] Device C8:12:C5:08:86:E1 Zephyr
+   [bluetooth]# scan on
+   Discovery started
+   [NEW] Device DC:98:FB:22:CA:3A Zephyr
+
+When started, ``bluetoothctl`` shows all BLE (and likely, BT/EDR) devices it
+knows about. As discussed above, the IPSP uses static random addresses, so
+entries like above accumulate and become stale. You need to be extra careful
```

Comment:
Never saw such a thing with qemu, perhaps this is a board thing that keeps rotating the address, due to restart? Or perhaps you had connected to each instance of those? Note static random addresses are supposed to be persistent, it is only the so-called private, resolvable and non-resolvable, random address that can change.

Reply:
> Or perhaps you had connected to each instance of those?

I guess that's it after - those aren't just discovered, but actually connected to addresses, and I had quite a bunch of those accumulated after 2 weeks of testing this stuff. I updated the wording correspondingly.